### ISSUE TITLE
Maria DB is no longer supported

### ISSUE THREAD

**Issue by balteravishay** [Azure-Samples/ctfd-azure-paas#18]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>
Deploy to Azure fails because Maria DB is no longer supported
### Any log messages given by the failure
>

### Expected/desired behavior
>
Deployment should work. Maria DB should be replaced with the new MySQL offering
### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by mateopc** [comment]
Hello,
I'm trying to deploy ctfd on Azure for an event in June. MariaDB is no longer supported and I'd be very interested in having a version of the deployment with MySQL. I'm going to try to make some changes on my side but I hope that a solution will be proposed soon.
Thanks a lot!

**Comment by sei-jfrederick** [comment]
Hello,

I'm in the same situation as mateopc.  Any help would be greatly appreciated!

**Comment by balteravishay** [comment]
@mateopc and @jjf205 , this is now fixed :)

**Comment by balteravishay** [comment]
fixes with #20

**Comment by sei-jfrederick** [comment]
Thank you!  I'll give it a go!
